.github/workflows/release-drafter.yaml

```diff
@@ -3,7 +3,10 @@ name: Release Drafter
 on:
   push:
     branches:
-      - releases/*
+      # we only want to draft release notes for the base version,
+      # e.g. for releases/v0.18, we draft v0.18.0 for all RCs
+      # this means that we explicitly don't want to draft release notes for releases/v0.18.0 or releases/v0.18.1
+      - 'releases/v[0-9]+.[0-9]+'
```

Comment:
This matches `releases/v10.0.3`. Use `releases/v[0-9]+.[0-9]+$`

Reply:
because the last `[0-9]+` is not a `.` this expression should also work here

Reply:
Huh 😱 Okay. :D